gmail/gmail_endpoint.bal

```diff
@@ -336,7 +330,7 @@ public remote function Client.listMessages(string userId, MsgSearchFilter? filte
         string uriParams = "";
         //The default value for include spam trash query parameter of the api call is false
         //If append unsuccessful throws and returns error
-        uriParams = check appendEncodedURIParameter(uriParams, INCLUDE_SPAMTRASH, <string>filter.includeSpamTrash);
+        uriParams = check appendEncodedURIParameter(uriParams, INCLUDE_SPAMTRASH, string.convert(filter.includeSpamTrash));
```

Comment:
Shall we use `io:sprintf` instead of `string.convert`? ``string.convert` will be removed most probably.